salt/elasticsearch/init.sls

```diff
@@ -55,35 +60,22 @@ set jvm maximum heap size:
         script.allowed_contexts: ingest
         # https://www.elastic.co/guide/en/elasticsearch/reference/7.10/bootstrap-checks.html
         discovery.type: single-node
-        {% if 'allowed_origins' in pillar.elasticsearch %}
-        # https://www.elastic.co/guide/en/elasticsearch/reference/7.10/modules-http.html
-        http.cors.enabled: 'true'
-        http.cors.allow-origin: "'{{ pillar.elasticsearch.allowed_origins }}'"
-        http.cors.allow-methods: OPTIONS, GET, POST
-        http.cors.allow-headers: X-Requested-With, Content-Type, Content-Length, Authorization
-        {% endif %}
     - separator: ': '
     - append_if_not_found: True
     - watch_in:
       - service: elasticsearch
 
-/etc/elasticsearch/jvm.options.d/bootstrap-checks.options:
-  file.managed:
-    - name: /etc/elasticsearch/jvm.options.d/bootstrap-checks.options
-    - contents: "-Des.enforce.bootstrap.checks=true"
-    - watch_in:
-      - service: elasticsearch
-{% else %}
-/etc/elasticsearch/elasticsearch.yml:
+/etc/elasticsearch/elasticsearch.yml disable cluster mode:
```

Comment:
Instead of a new state, we can just make `file.comment` a second instruction in the above state.

Reply:
We cannot merge states with the same type, so because these actions are both `file.*` it errors.